core/src/Kokkos_View.hpp

```diff
@@ -1139,9 +1152,8 @@ class View : public ViewTraits<DataType, Properties...> {
       typename std::enable_if<(Kokkos::Impl::are_integral<I0, Args...>::value &&
                                (1 == Rank) && !is_default_map),
                               reference_type>::type
-      access(const I0& i0, Args... KOKKOS_IMPL_SINK(args)) const {
-    KOKKOS_IMPL_VIEW_OPERATOR_VERIFY(
-        KOKKOS_IMPL_SINK((m_track, m_map, i0, args...)))
+      access(const I0& i0, Args... args) const {
```

Comment:
this doesn't sink args away anymore, so we will get unused parameter warnings.

Reply:
Sounds like we need a build with debug bounds check enabled

Reply:
Before `ARG` was not used in `KOKKOS_IMPL_VIEW_OPERATOR_VERIFY` but now it is and we handle all the parameters in `verify_space::check` (where only `tracker` is ever used) so I think this is correct. Actually, we have https://github.com/kokkos/kokkos/blob/03ded4e816da5927b946065b005ebfdf38e5a438/.jenkins#L211-L247 that enables `Kokkos_ENABLE_DEBUG_BOUNDS_CHECK`.